pandas/core/generic.py

```diff
@@ -8982,13 +8983,16 @@ def where(
 
             - 'raise' : allow exceptions to be raised.
             - 'ignore' : suppress exceptions. On error return original object.
-
         try_cast : bool, default False
             Try to cast the result back to the input type (if possible).
+        invert : bool, default False
+            Whether or not to invert `cond`. This is done after alignment and
+            filling any missing values with `False`.
```

Comment:
.. versionadded:: 1.2

and maybe a bit more about it (and an example?) in whatsnew.

Reply:
see my comments above, no need to add to the public api